snooty/rstparser.py

```diff
@@ -486,6 +486,45 @@ def run(self) -> List[docutils.nodes.Node]:
         return [node]
 
 
+class ListTableDirective(docutils.parsers.rst.Directive):
```

Comment:
I don't think you need this anymore; the default directive handler (as created by the `rstspec.toml`) should be okay I think? Try getting rid of this class and see what happens.

Reply:
Works!